indico/modules/events/abstracts/templates/management/abstract_list.html

```diff
@@ -170,6 +170,13 @@
                             data-ajax-dialog>
                         {%- trans %}Author list{% endtrans -%}
                     </button>
+                    <div id="email-abstract-roles-container"
```

Comment:
The Email button isn't aligning/spacing properly with the other buttons - I think this is because the `<button>` gets rendered inside this `<div>`. It should either be at the same level with the others, or in a `<div class="group">` with the author list.

<img width="983" alt="image" src="https://user-images.githubusercontent.com/27357203/206202829-ab0c6a75-9e96-4cb9-847c-7de6e65c2e39.png">

A quick workaround I found is to add the class `group` here. One-button groups seem to work fine.